libs/books/feature/src/lib/book-search/book-search.component.html

```diff
@@ -6,22 +6,23 @@
       type="search"
       placeholder="Search for books to add to your reading list"
       formControlName="term"
+      autocomplete="off"
     />
-    <button mat-icon-button matSuffix>
+    <button mat-icon-button matSuffix aria-label="search">
       <mat-icon>search</mat-icon>
     </button>
   </mat-form-field>
 </form>
 
 <ng-container *ngIf="searchTerm; else empty">
   <div class="book-grid">
-    <div class="book" data-testing="book-item" *ngFor="let b of books">
+    <div class="book" data-testing="book-item" *ngFor="let b of books | async">
       <div class="book--title">
-        {{ b.title }}
+        <div role="heading" aria-level="1">{{ b.title }}</div>
```

Comment:
Please use optional chain for all properties in HTML templates

eg. `b?.title`